modules/UI/shared_video/SharedVideoThumb.js

```diff
@@ -1,10 +1,15 @@
-/* global $ */
+/* global $, APP */
 
-import Logger from 'jitsi-meet-logger';
+/* eslint-disable no-unused-vars */
```

Comment:
We don't use the react eslint config for the legacy code and this leads to ignoring the jsx expressions which leads to eslint complaining about the imports for the components. Probably we should've fix this but especially in this case I don't see a point of spending time on it since this file will be deleted in my next PR...

Reply:
No worries, thanks for explaining!